doc/nrf/ug_multi_image.rst

```diff
@@ -182,6 +180,54 @@ This means that to run menuconfig, for example, you invoke the ``menuconfig`` ta
 You can also invoke any child target directly from its build directory.
 Child build directories are located at the root of the parent's build directory.
 
+Controlling the build process
+=============================
+
+The child image is built using CMake's built-in build command ``cmake --build``.
+This mechanism allows additional control of the build process through CMake.
+
+CMake options
+-------------
+
+The following CMake options are propagated from the CMake command of the parent image to the CMake command of the child image:
+
+* ``CMAKE_BUILD_TYPE``
+* ``CMAKE_VERBOSE_MAKEFILE``
+
+You can add other CMake options to a specific child image in the same way as you can set `image-specific variables <Image-specific variables>`_.
+For example, add ``-Dmcuboot_CMAKE_VERBOSE_MAKEFILE`` to the parent's CMake command to build the ``mcuboot`` child image with verbose output.
+
+To enable additional debug information for the multi-image build command, set the CMake option ``MULTI_IMAGE_DEBUG_MAKEFILE`` to the desired debug mode.
```

Comment:
Correct that this is a CMake option and not an environment variable?

Reply:
That is correct.